napari/_vispy/visuals/util.py

```diff
@@ -0,0 +1,11 @@
+class TextureMinix:
```

Comment:
```suggestion
class TextureMixin:
```

(And obviously the usages need to change too.)

Could you add a docstring that explains what this class does?

Reply:
Is docstring ok now?